### ISSUE TITLE
After latest Twitter changes, I don't think Tweet is working anymore.

### ISSUE THREAD

**Issue by lifeboatpres** [face-hh/tweetfree#2]
This is really cool code!

I don't think Tweet is working anymore after the latest Twitter changes. On my code, I was unable to get page.keyboard.type or page.type to work at all for Tweets and had to switch to page.keyboard.sendCharacter to send the Tweet one character at a time.

**Comment by face-hh** [comment]
Yeah... I've seen the latest Twitter changes. If you can, please open a pull request with the fix that worked for you, that would be awesome :)

**Comment by lifeboatpres** [comment]
I never tried your code as it didn't look robust enough to replace my code which unfortunately wasn't working after the change either. (Your code is based on exact order for a div, etc.)

Here's my new code for posting a message:

```
for (c of message+" "+post_url)
{await page.keyboard.sendCharacter(c)}
```
Note that there are some cases where page.keyboard.sendCharacter() might work when page.keyboard.type() doesn't. This usually happens in situations where the target input field has event listeners attached to keydown or keyup events that either prevent the default action or stop the event propagation.

Also, after the big Twitter changes, all my Linode machines were unable to run my script as Twitter was giving "try again" errors, etc. as if it was overloaded so I had to run the script on my home machine. (So Twitter didn't like the IPs of my Linode machines.) I noticed yesterday that my Linode machines had access to Twitter again.
